docs/README.md

```diff
@@ -1,12 +1,35 @@
 # Introduction
 
-An **in-toto attestation** is authenticated metadata about one or more
-software artifacts[^1]. The intended consumers are automated policy engines,
-such as [in-toto-verify] and [Binary Authorization].
+The **in-toto Attestation Framework**
 
-**TODO: Clean this up; add a docs index.**
+1.  defines a standard [format](../spec/v1.0-draft/) for attestations which
+    bind subjects, the artifacts being described, to arbitrary authenticated
+    metadata about the artifact
+2.  provides a set of [pre-defined statements](../spec/predicates/) for
+    communicating authenticated metadata throughout and across software supply
+    chains
 
-Examples of hypothetical attestations:
+The framework was designed to support use of the data by automated policy
+engines, such as [in-toto-verify] and [Binary Authorization]. However, any
+system that creates or consumes verifiable claims about how a piece of software
+is produced can benefit from the in-toto Attestation Framework.
+
+## Project goals
+
+-   Standardize artifact metadata without being specific to the producer or
+    consumer. This way CI/CD pipelines, vulnerability scanners, and other
+    systems can generate a single set of attestations that can be consumed by
+    anyone, such as [in-toto] or [Binary Authorization].
```

Comment:
This repeats the last paragraph in the previous section. Maybe merge them?

Reply:
I'll remove the "such as in-toto or Binary Authorization" but leave the rest as-is. We could certainly clean this up further, but I like the current split between design intent and design goals. WDYT?